packages/loader/container-loader/src/containerContext.ts

```diff
@@ -208,6 +209,8 @@ export class ContainerContext implements IContainerContext {
 		this._fluidModuleP = new LazyPromise<IFluidModuleWithDetails>(async () =>
 			this.loadCodeModule(_codeDetails),
 		);
+
+		this.supportedFeatures = new Map([["referenceSequenceNumbers", true]]);
```

Comment:
Some inline comment on what that feature means and how runtime can leverage that would be nice